docs/docker-dev/docker-dev-setup-manual.md

```diff
@@ -0,0 +1,18 @@
+# Docker Development Environment Setup
+
+1. Install [Docker](https://docs.docker.com/get-docker/) if not already.
+2. On the terminal, run command `curl -o- https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/main/dev-tools/install-docker-dev.sh | bash`. This should create a folder named `opensearch-dashboards-docker-dev` and it should contain two files: `docker-compose.yml` and `entrypoint.sh`.
+3. Open [VsCode](https://code.visualstudio.com/download). Install here if not already. Make sure VsCode has the extension `Dev Containers` and `Docker` installed. If not, go to Extensions tab, search and install them.
+4. Under the Discover tab, click `Open Folder`, and open the `opensearch-dashboards-docker-dev` folder that we just created.
+5. In the workspace folder, set environment variable for the fork repository URL: run `export REPO_URL=[insert your fork repo url here]`. If fork repo has not been created: Go to [OpenSearch Dashboards github page](https://github.com/opensearch-project/OpenSearch-Dashboards) and under fork, select create a new fork, and then copy the https link of the fork url and use it in the above command.
```

Comment:
Where/how should they get the fork repo URL? Can they find it from within VS Code?

Reply:
Also, is this necessary every time they start the container, or is it just for initial setup? Any ways we could automate it?

Reply:
yea this step for setting up environment variable is necessary every time when they re-start the compose file in a new terminal. Will add a sentence specifying this.

I have explained how to get the fork repo URL in the second part of step 5.